plugins/adr/src/components/AdrReader/AdrReader.tsx

```diff
@@ -46,12 +46,17 @@ export const AdrReader = (props: {
   const adrApi = useApi(adrApiRef);
   const adrLocationUrl = getAdrLocationUrl(entity, scmIntegrations);
   const adrFileLocationUrl = getAdrLocationUrl(entity, scmIntegrations, adr);
+  const discoveryApi = useApi(discoveryApiRef);
 
   const { value, loading, error } = useAsync(
     async () => adrApi.readAdr(adrFileLocationUrl),
     [adrFileLocationUrl],
   );
 
+  const { value: backendUrl } = useAsync(
```

Comment:
We probably want to wait on the loading property from this before rendering content since we depend on this to be defined and if this fails for whatever reason then we should report that as well since that will prevent images from loading correctly